python/plugins/processing/core/ProcessingConfig.py

```diff
@@ -83,6 +84,10 @@ def initialize():
             ProcessingConfig.tr('Use only selected features'), True))
         ProcessingConfig.addSetting(Setting(
             ProcessingConfig.tr('General'),
+            ProcessingConfig.IGNORE_NULL_GEOMS,
+            ProcessingConfig.tr('Ignore features with null geometries'), True))
```

Comment:
On the other hand any algorithm that handles null geoms correctly nowadays will perform worse after this patch and needs to be updated.

What's the downside of my proposed approach? It forces developers and users to *think* about null values instead of at any cose producing (potentially wrong) results.